v3/cypress/e2e/table.spec.ts

```diff
@@ -1145,6 +1145,7 @@ context("case table ui", () => {
 
       cy.log("click Set Color button to dismiss color palette and change color")
       cpp.getSetColorButton().click()
+      cy.wait(1000) // Wait for the color change to be reflected
```

Comment:
Could we get by with `100` here like with other waits in this test? Surely, it can't take a full second for the color change to be reflected.

Reply:
I tried 500 and it was still failing 😞